doc/kernel/other/ring_buffers.rst

```diff
@@ -144,6 +194,42 @@ can be specified.
 	...
     }
 
+Bytes are copied to a ring buffer by calling
+:cpp:func:`sys_ring_buf_bytes_cpy_put()`.
+
+.. code-block:: c
+
+    u8_t my_data[MY_RING_BUF_BYTES];
+    u32_t ret;
+
+    ret = sys_ring_buf_bytes_cpy_put(&ring_buf, my_data, SIZE_OF(my_data));
+    if (ret != SIZE_OF(my_data)) {
+        /* not enough room, partial copy. */
+	...
+    }
+
+Data can be added to a ring buffer by direct operation on a ring buffer memory.
```

Comment:
How about:

    Data can be added to a **byte mode** ring buffer by directly access the
    ring buffer's memory.  For example: